### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by silviatofana**
In this milestone, I implemented the following:

Build interactive console app
Create a main.rb file that will serve as your console app entry-point. It should have methods that do the following:
. List all books.
. List all people.
. Create a person (teacher or student, not a plain Person).
. Create a book.
. Create a rental.
. List all rentals for a given person id.
In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
. Present the user with a list of options to perform.
. Lets users choose an option.
. If needed, ask for parameters for the option.
. Have a way to quit the app.
Kindly review my code
Thank you!

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    id = gets.chomp
+
+    puts 'Rentals: '
+
+    @rentals.each do |rental|
+      puts "Date: #{rental.date}, Book: '#{rental.book.title}' by #{rental.book.author}" if rental.person.id == id.to_i
+    end
+    choices
+  end
+end
```
Comment (julie-ify) [inline main.rb]: - [ ] According to the requirements, you are expected to create a separate `app.rb` file that will contain the App class you defined above. It should contain the methods I selected here.

  Basically, this `main.rb` file should only contain code related to listing the choices and running the program. Any other method for the logic of listing and creation of the books, person, or rental (like these ones) should be in the App file.

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rental_by_person_id
+    else
+      puts 'Exit'
+    end
+  end
```
Comment (julie-ify) [inline main.rb]: - [ ] Only this code that serves as the entry point and the code in the last lines of this file for starting the app should be in this `main.rb` file

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @silviatofana,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (mugishasam123) [inline main.rb]: - [ ] Please, as suggested by the previous reviewer you should create an app file that contains all logic and methods to be used throughout the app and include that app file in the `main.rb` file to make your app structure professional. you got this  👍

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @silviatofana  👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
alt


 ## Highlights
- No linter errors 🚀 
- Nice use of git-flow ✔️ 
- Good commit messages ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @silviatofana,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by silviatofana** [comment]
thanks

**Pull request merged**